### ISSUE TITLE
Auto-completed component imports sometimes end up in the wrong place

### ISSUE THREAD

**Issue by mpbarlow** [vuejs/vetur#2496]
- [x] I have searched through existing issues
- [X] I have read through [docs](https://vuejs.github.io/vetur)
- [X] I have read [FAQ](https://github.com/vuejs/vetur/blob/master/docs/FAQ.md)
- [X] I have tried restarting VS Code or running `Vetur: Restart VLS`

## Info

- Platform: macOS (x86_64)
- Vetur version: 0.30.2
- VS Code version: 1.51.1

## Problem

Auto-completed imports are spliced into a random part of the file about 50% of the time.

By "auto-completed imports", I mean where you start typing the name of a component in the template (that has not yet been imported into the file), and hit return. The `.vue` file is imported, and the component is added to the `components` block. However, while the `import` statement always ends up in the right place, the component often ends up somewhere totally random.

Two screenshots:

![Screenshot 2020-11-25 at 12 46 19](https://user-images.githubusercontent.com/26557284/100229746-9f1d0100-2f1c-11eb-9121-818c796c322b.png)
![Screenshot 2020-11-24 at 13 43 05](https://user-images.githubusercontent.com/26557284/100229727-99272000-2f1c-11eb-941a-4d67d5bbf90e.png)

Here you see that the name of the component has been added after the closing `script` tag, and in the middle of the `data` function, respectively.

## Reproducible Case

I have unfortunately not been able to create a concrete reproduction of this, but it seems to be about 50/50. Even within the same file, I can import four components back to back, and two might be fine, two might be in the wrong place.

**Comment by yoyo930021** [comment]
If you can create a reproducible project, I can handle it better.
Even if it's only the first meeting.

**Comment by mpbarlow** [comment]
While it's happening regularly on a few different projects I'm working on, I can't make it happen at all on a completely fresh project. As such I think it's likely a clash with some other VS Code extension, rather than a problem with Vetur.

**Comment by Gameghostify** [comment]
I have this issue as well. @mpbarlow if you still have the issue, would you mind sharing what extensions you have installed? I would then check if any of my extensions matched yours, maybe we can find out what's causing this.

It doesn't happen for me as often, but still 1/5 times

**Comment by Severino** [comment]
I have encountered this issue on various systems. Never had a similar issue with any other plugin. I tested it without any extension except for vetur installed. I tried to put together a minimal test case, but it happens totally unpredictably!

**When does it happen:**
Always when using IntelliJ in the template block.

**How does it happen:**
1. Insert does not insert comma at the end of last element: {first\nSecond,}
2. Insert does happen inside the next variable name in the component: e.g. methMYCOMPONENTod:{ }
3. Insert happens in next Object block separating e.g. a function with a comma and inserting the component after: sort, MYCOMPONENT(){*

**Here is a small gif how it happens:**
_Note: The fourth component get's inserted into the methods block after the method first_

![live_short](https://user-images.githubusercontent.com/7532600/147467404-06428ba9-2410-47cf-acf8-ca62bc25d363.gif)


It drives me crazy!
